csv/scanner.go

```diff
@@ -1,11 +1,25 @@
+// Package csv scans CSV files, provides easy access to individual
+// columns, and can also read field values into a struct (analogous
+// to unmarshaling JSON or XML).
+//
+// It thinly wraps the standard library's [csv.Reader] and exposes
+// most of its configuration "knobs" and behavior.  Knowledge of
+// the csv.Reader will help in configuring and running these
+// scanners.
+//
+// Advance the scanners with the Scan method and check errors with
+// the Error method (unlike fields and fixedwidth, which use Err).
 package csv
 
 import (
 	"encoding/csv"
 	"io"
 )
 
-// Scanner wraps a csv.Reader via an API similar to that of bufio.Scanner.
```

Comment:
The note about bufio.Scanner got moved to the top-level scanners.go [doc string](https://github.com/smartystreets/scanners/pull/7/files#diff-d79e5d1d5bdbf80413e72607249dd87ba03228ca459f796ef7e385fb366f5914R4).